exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,105 @@
+    },
+    {
+      "description": "fourth day four calling birds",
+      "property": "verse",
+      "input": 4,
+      "expected": "On the fourth day of Christmas my true love gave to me, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "fifth day five gold rings",
+      "property": "verse",
+      "input": 5,
+      "expected": "On the fifth day of Christmas my true love gave to me, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "sixth day six geese-a-laying",
+      "property": "verse",
+      "input": 6,
+      "expected": "On the sixth day of Christmas my true love gave to me, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "seventh day seven swans-a-swimming",
+      "property": "verse",
+      "input": 7,
+      "expected": "On the seventh day of Christmas my true love gave to me, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "eighth day eight maids-a-milking",
+      "property": "verse",
+      "input": 8,
+      "expected": "On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "ninth day nine ladies dancing",
+      "property": "verse",
+      "input": 9,
+      "expected": "On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "tenth day ten lords-a-leaping",
+      "property": "verse",
+      "input": 10,
+      "expected": "On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "eleventh day eleven pipers piping",
+      "property": "verse",
+      "input": 11,
+      "expected": "On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree."
+    },
+    {
+      "description": "twelfth day twelve drummers drumming",
+      "property": "verse",
+      "input": 12,
```

Comment:
Instead of `"input"` how about `"number"` since it is the verse number being asked for.